modules/core/24-host/v2/packet_keys.go

```diff
@@ -6,25 +6,24 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// PacketReceiptKey returns the store key of under which a packet
-// receipt is stored
-func PacketReceiptKey(channelID string, sequence uint64) []byte {
-	return []byte(fmt.Sprintf("receipts/channels/%s/sequences/%s", channelID, sdk.Uint64ToBigEndian(sequence)))
-}
-
-// PacketAcknowledgementKey returns the store key of under which a packet acknowledgement is stored.
-func PacketAcknowledgementKey(channelID string, sequence uint64) []byte {
-	return []byte(fmt.Sprintf("acks/channels/%s/sequences/%s", channelID, sdk.Uint64ToBigEndian(sequence)))
+// PacketCommitmentKey returns the store key of under which a packet commitment is stored.
+func PacketCommitmentKey(channelID string, sequence uint64) []byte {
+	return append(append([]byte(channelID), byte(1)), sdk.Uint64ToBigEndian(sequence)...)
 }
 
 // PacketCommitmentPrefixKey returns the store key prefix under which packet commitments for a particular channel are stored.
 func PacketCommitmentPrefixKey(channelID string) []byte {
```

Comment:
though wouldn't this key better belong in `channel/v2/types/keys.go`? Its only used for iterating through commitments in grpc req?

Currently similar for v1 but maybe a good idea to move. Leaving as is for now.